packaging/docker/health.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+#
+# This is the script that gets run for our Docker image health checks.
+
+if [ -z "${NETDATA_HEALTH_CHECK}" ] ; then
+    # If users didn't request something else, query `/api/v1/info`.
+    NETDATA_HEALTH_CHECK="http://localhost:19999/api/v1/info"
+fi
+
+case "${NETDATA_HEALTH_CHECK}" in
+    cli)
+        netdatacli ping || exit 1
+        ;;
+    *)
+        curl -sS "${NETDATA_HEALTH_CHECK}" || exit 1
```

Comment:
Please use `-q` as the first argument. And add `-L` in case of redirects.

It would be better to also rename `NETDATA_HEALTH_CHECK` to `NETDATA_HEALTH_CHECK_URL` to be more clear.

Reply:
Not using `-q` was intentional here. Docker stores the health check output and you can inspect it (`docker inspect --format "{{json .State.Health }}" <container name> | jq`), which is useful for figuring out why it's failing in a case like this.

Agreed on the `-L` though.